dask/blockwise.py

```diff
@@ -963,15 +971,15 @@ def make_blockwise_graph(
             deps.update(func_future_args)
             args += list(func_future_args)
 
-        if deserializing and not concatenate and isinstance(func, bytes):
+        if deserializing and isinstance(func, bytes):
             # Construct a function/args/kwargs dict if we
             # do not have a nested task (i.e. concatenate=False).
             # TODO: Avoid using the iterate_collection-version
             # of to_serialize if we know that are no embeded
             # Serialized/Serialize objects in args and/or kwargs.
             if kwargs:
                 dsk[out_key] = {
-                    "function": dumps_function(apply),
+                    "function": dumps_function(func),
```

Comment:
If `isinstance(func, bytes)`, do we want to `dumps_function` it? Isn't it already dumped?

Reply:
Weird - I could have sworn I removed that `dumps_function`.  Thanks for pointing that out!